web/modules/custom/portland/modules/portland_address_verifier/src/Plugin/WebformElement/PortlandAddressVerifier.php

```diff
@@ -27,43 +27,6 @@
  */
 class PortlandAddressVerifier extends WebformCompositeBase {
 
-  /**
-   * {@inheritdoc}
-   */
-  public function getInfo() {
-    return parent::getInfo() + [
-      '#pre_render' => [
-        [get_class($this), 'preRenderCompositeElement'],
-      ],
-    ];
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public static function preRenderCompositeElement($element) {
-    $element['#element_validate'][] = [get_class(), 'validateMyCompositeElement'];
-    return $element;
-  }
-
-  /**
-   * Custom validation handler.
-   */
-  public static function validateMyCompositeElement(&$element, FormStateInterface $form_state, &$complete_form) {
-    // Check if the element is visible based on your conditional logic.
-    $visible = TRUE; // Replace this with your actual visibility check.
```

Comment:
These 3 unused functions were removed. This is hard-coded to true so it wasn't ever running.